### ISSUE TITLE
Poll: SQL Queries

### ISSUE THREAD

**Issue by herrfugbaum** [herrfugbaum/qsv#3]
Which SQL Query do you want to see implemented next?

[![](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/WHERE)](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/WHERE/vote)
[![](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/DISTINCT)](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/DISTINCT/vote)
[![](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/ORDER%20BY)](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/ORDER%20BY/vote)
[![](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/INSERT)](https://api.gh-polls.com/poll/01CWPKV0RQ0MRQNAEA9ZFB4B8H/INSERT/vote)

**Comment by herrfugbaum** [comment]
Note: If you want to contribute, the SQL parser [q](/herrfugbaum/q) is located in a second repository. 
It's based on chevrotrain and follows their [tutorial](http://sap.github.io/chevrotain/docs/tutorial/step0_introduction.html).

**Comment by herrfugbaum** [comment]
The ORDER BY clause is already implemented in [qsv](/herrfugbaum/qsv/blob/master/src/repl.js#L15) and q [1](https://github.com/herrfugbaum/q/blob/master/lexer.js#L29), [2](https://github.com/herrfugbaum/q/blob/master/parser.js#L73), [3](https://github.com/herrfugbaum/q/blob/master/cst-visitor.js#L78) but it lacks support for multiple columns.
So it's possible to query like
```sql
SELECT * FROM table ORDER BY name
````
or
```sql
SELECT * FROM table ORDER BY name DESC
````
but not like

```sql
SELECT * FROM table ORDER BY name DESC, age
````

**Comment by herrfugbaum** [comment]
LIMIT is implemented with #7
